lively.morphic/layout.js

```diff
@@ -840,15 +858,27 @@ export class TilingLayout extends Layout {
       morph.withMetaDo({ isLayoutAction: true, skipRender: true }, () => morph.position = pt(newPosX, newPosY));
     }
 
-    if (widthPolicy === 'fill' && String(newWidth) !== 'NaN' && newWidth !== morph.width) {
-      morph.withMetaDo({ isLayoutAction: true, skipRender: false }, () => {
-        morph.width = newWidth;
-      });
+    let retriggerLayout = false;
+
+    if (!isPreliminary && widthPolicy === 'fill' && String(newWidth) !== 'NaN' && newWidth !== morph.width) {
+      const widthBefore = morph.width;
+      morph.withMetaDo({ isLayoutAction: true, skipRender: false }, () => morph.width = newWidth);
+      if (morph.isText && !morph.fixedHeight) {
+        retriggerLayout = true;
+      }
+      if (morph.isText && !morph.fixedWidth) {
+        console.warn(`The text morph ${morph.id} is set to hug its contents yet configure to fill the container horizontally!`);
+      }
```

Comment:
This one gets really spammy, even when only playing around for a bit on the new branch...

Reply:
Which goes to show that there are a lot of nonsensical configurations in the system. Would you prefer a default behavior? If so which one should win?